### PR TITLE
Update api-hello-world.md

### PR DESCRIPTION
**Pull request by msbemba**
Added URL - Documentation link to the PS script. It needs to be a complete link as it is hashed

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/10173

@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/api-hello-world.md [hunk 1]
```diff
@@ -158,7 +158,7 @@ Done! You have successfully registered an application!
   $dateTime = (Get-Date).ToUniversalTime().AddHours(-48).ToString("o")
 
   # The URL contains the type of query and the time filter we create above
-  # Read more about other query options and filters at   Https://TBD- add the documentation link
+  # Read more about other query options and filters at https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/get-alerts
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/api-hello-world.md:161]:
```suggestion
  # Read more about [other query options and filters](/microsoft-365/security/defender-endpoint/get-alerts)
```

Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/api-hello-world.md]:
```suggestion
  # Read more about other query options and filters at https://learn.microsoft.com/microsoft-365/security/defender-endpoint/get-alerts
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Pull request merged**